### PR TITLE
block_hotplug: Add a case that hotplug a max size disk

### PR DESCRIPTION
**Pull request by YongxueHong**
Add a case that hotplug a max image size disk with qcow2 and raw.

ID: 1564836
Signed-off-by: Yongxue Hong <yhong@redhat.com>

### CODE REVIEW

**Comment by YongxueHong** [comment]
Depends on https://github.com/avocado-framework/avocado-vt/pull/2137

**Comment by YongxueHong** [comment]
@phou if you are free, please me review it, thanks.

**Comment by hereischen** [comment]
Acked-by: Haotong Chen hachen@redhat.com

File: qemu/tests/block_hotplug.py [hunk 1]
```diff
@@ -64,6 +68,40 @@ def run_sub_test(params, plug_tag):
...
+        """
+        Get the disk size from guest.
+
+        :param did: the disk of id, e.g. sdb,sda for linux, 1, 2 for windows
+        :return: the disk size
+        """
+        if params['os_type'] == 'linux':
+            size = utils_disk.get_linux_disks(session)[did][1].strip()
+        else:
+            random_str = random.sample(string.ascii_letters + string.digits, 4)
```
Comment (hereischen) [inline qemu/tests/block_hotplug.py]: It seems vt has a wrapper for  random_str, please have a look, but this is fine for me

Comment (YongxueHong) [reply qemu/tests/block_hotplug.py]: Thanks, will update it.

**Comment by zhenyzha** [comment]
Tested 
"block_hotplug.default.block_scsi.fmt_raw.max_size.with_plug.default.one_pci" 
"block_hotplug.default.block_scsi.fmt_qcow2.max_size.with_plug.default.one_pci" 
"block_hotplug.default.block_virtio.fmt_raw.max_size.with_plug.default.one_pci" 
"block_hotplug.default.block_virtio.fmt_qcow2.max_size.with_plug.default.one_pci" 
 on RHEL.7.7 guest, all passed.
 
(3/3) Host_RHEL.m7.u7.raw.virtio_scsi.up.virtio_net.Guest.RHEL.7.7.ppc64le.io-github-autotest-qemu.block_hotplug.block_scsi.fmt_raw.max_size.with_plug.one_pci: PASS (99.18 s)
 (1/1) Host_RHEL.m7.u7.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.7.7.ppc64le.io-github-autotest-qemu.block_hotplug.block_scsi.fmt_qcow2.max_size.with_plug.one_pci: PASS (120.00 s)
 (3/3) Host_RHEL.m7.u7.raw.virtio_blk.up.virtio_net.Guest.RHEL.7.7.ppc64le.io-github-autotest-qemu.block_hotplug.block_virtio.fmt_raw.max_size.with_plug.one_pci: PASS (100.54 s)
 (3/3) Host_RHEL.m7.u7.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.7.7.ppc64le.io-github-autotest-qemu.block_hotplug.block_virtio.fmt_qcow2.max_size.with_plug.one_pci: PASS (107.57 s)

Acked-by: Zhenyu Zhang <zhenyzha@redhat.com>

**Comment by peixiu** [comment]
Tested with new code, passed.
 (1/4) Host_RHEL.m8.u1.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.block_hotplug.block_scsi.fmt_qcow2.with_plug.one_pci.q35: PASS (88.30 s)
 (2/4) Host_RHEL.m8.u1.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.block_hotplug.block_scsi.fmt_qcow2.max_size.with_plug.one_pci.q35: PASS (66.34 s)
 (3/4) Host_RHEL.m8.u1.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.block_hotplug.block_scsi.fmt_raw.with_plug.one_pci.q35: PASS (100.45 s)
 (4/4) Host_RHEL.m8.u1.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.block_hotplug.block_scsi.fmt_raw.max_size.with_plug.one_pci.q35: PASS (65.76 s)

Acked-by: Peixiu Hou phou@redhat.com

**Pull request merged**